<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileList</class>
 <widget class="QWidget" name="fileList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gallery</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>720</width>
     <height>1180</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>1280</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="listShow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>1200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
